frontend/coprs_frontend/coprs/forms.py

```diff
@@ -655,6 +656,17 @@ class CoprForm(BaseForm):
         filters=[StringListFilter(), StripUrlSchemaListFilter()],
         validators=[wtforms.validators.Optional()],)
 
+    repo_priority = wtforms.IntegerField(
+        "The priority value of this repository",
+        description="""If there is more than one package to install from (e.g. vim from
+        Fedora repo and Copr repo), the repo with lower priority is picked.""",
```

Comment:
The `man dnf.conf` says one important thing that might be surprising for some users

> repo with the lowest priority value is picked, possibly despite being less convenient otherwise (e.g. **by being a lower version**).

Can we say something like this also? I think maybe even copy-pasting the whole `priority` description from `man dnf.conf` might be a good idea.